spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -13,17 +13,17 @@
   end
 
   it 'finds context without `when` at the beginning' do
-    expect_offense(<<-RUBY)
+    expect_offense(<<-'RUBY')
       context 'the display name not present' do
-              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Context description should match /^when\\b/, /^with\\b/, or /^without\\b/.
+              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Context description should match /^when\b/, /^with\b/, or /^without\b/.
```

Comment:
There is a difference at a glance. Wondering how it turns out that both work :thinking: 
```
<<~A
/^when\b/
A
=> "/^when\b/\n"
<<~'A'
/^when\b/
A
=> "/^when\\b/\n"
```
Maybe there's some escaping logic in `expect_offense`? I can't find any code in `expect_offense` and there are no specs for this method or `AnnotatedSource` in `rubocop` specs.

Reply:
The argument of `expect_offense` is a heap document, so a heap document enclosed in single quotes is completely a string as written. So the following would be the same.

```ruby
irb(main):001:0" <<~A
irb(main):002:0" /^when\\b/
irb(main):003:0> A
=> "/^when\\b/\n"
irb(main):004:0' <<~'A'
irb(main):005:0' /^when\b/
irb(main):006:0> A
=> "/^when\b/\n"
```

Am I missing something?

Reply:
Oh, please accept my apologies, I've missed the point that you've changed the contents of the string and removed unnecessary escaping.
Looks much better this way indeed!

Reply:
Please add to the PR description what the benefit/downside is of either syntax. I always forget which is which.